pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2266,17 +2287,22 @@ func DomainCreate(status types.DomainStatus) (int, string, error) {
 	filename := xenCfgFilename(status.AppNum)
 	mpFileName := mountPointFileName(status.AppNum)
 	log.Infof("DomainCreate %s ... xenCfgFilename - %s\n", status.DomainName, filename)
+	var containerImageSha256 string
+	for _, ds := range status.DiskStatusList {
+		if ds.Format == zconfig.Format_CONTAINER {
+			containerImageSha256 = ds.ImageSha256
+			continue
```

Comment:
In the old code you set containerImageSha256 based on the first container, and with this change you are setting it to the last container. Does that matter? Intentional?

Reply:
It doesn't matter because of manifest will either one or none container disks.

Reply:
OK